data/mkbhru.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Manish Kumar",
+  "type": "personal",
+  "bio": "A fullstack developer",
+  
```

Comment:
Looks great, this empty line will be fixed by github actions 👍

Reply:
thanks for accepting it :+1: